src/binder_util.c

```diff
@@ -815,6 +853,47 @@ binder_strv_from_hidl_string_vec(
     return NULL;
 }
 
+gboolean
+binder_read_string16_parse_int(
+    GBinderReader* reader,
+    gint32* value)
+{
+    /* Read a string and parse integer value from it */
+    gboolean ret = FALSE;
+    char* str = gbinder_reader_read_string16(reader);
+
+    if (str) {
+        ret = gutil_parse_int(str, 10, value);
+        g_free(str);
+    }
+    return ret;
+}
+
+char**
+binder_strv_from_string16_array(
+    GBinderReader* reader)
+{
+    if (reader) {
+        gint32 count;
+        gbinder_reader_read_int32(reader, &count);
+        if (count < 0) {
+            count = 0;
+        }
+        char** out = g_new(char*, count + 1);
```

Comment:
So, um, for clarity, why not use `g_new0()` so the latter `*ptr = NULL` could be removed?